js/renovation/ui/grids/data_grid_light/data_grid_light.tsx

```diff
@@ -26,18 +32,65 @@ export const viewFunction = (viewModel: DataGridLight): JSX.Element => (
   >
     <div className="dx-datagrid dx-gridbase-container" role="grid" aria-label="Data grid">
       <TableHeader columns={viewModel.props.columns} />
-      <TableContent columns={viewModel.props.columns} dataSource={viewModel.props.dataSource} />
+      <TableContent columns={viewModel.props.columns} dataSource={viewModel.visibleItems} />
+      <GridPager
+        allowedPageSizes={viewModel.props.pager.allowedPageSizes}
```

Comment:
Try to pass pager props vie single prop:
`pager={viewModel.props.pager}`

It will be less code in this case.

Reply:
I don't think that it's good to create new `@Nested` in pager, so I added spreading for pager props